addons/account/models/account_journal_dashboard.py

```diff
@@ -423,7 +423,14 @@ def create_cash_statement(self):
     def action_open_reconcile(self):
         if self.type in ['bank', 'cash']:
             # Open reconciliation view for bank statements belonging to this journal
-            bank_stmt = self.env['account.bank.statement'].search([('journal_id', 'in', self.ids)]).mapped('line_ids')
+            bank_stmt = self.env['account.bank.statement.line'].search([
+                ('journal_id', 'in', self.ids),
+                # take not reconciled lines only. See _check_lines_reconciled method
+                ('account_id', '=', False),
+                ('journal_entry_ids', '=', False),
+                ('amount', '!=', 0),
```

Comment:
```
                ('amount', '!=', 0),
```
is not exactly the same as ``currency_id.is_zero``. It may give more result, if amount is `0.001` somehow, while currency is rounded to two digits. But it should not be a problem and we can reconcile such almost zero value or temporary change rounding configuration.

https://github.com/odoo/odoo/blob/b27c0698d95cbccaa6c35a1fd3faa852a71ac2c2/addons/account/models/account_bank_statement.py#L132-L138